lib/vsc/administration/user.py

```diff
@@ -402,13 +413,12 @@ def __setattr__(self, name, value):
         super(VscTier2AccountpageUser, self).__setattr__(name, value)
 
 
-
 cluster_user_pickle_location_map = {
```

Comment:
these are module constants? move them up, and use all caps
also, this need some vub support too?

Reply:
I'm not sure actually. They are not used anywhere in vsc-administration. @itkovian ?

Reply:
@itkovian says this is legacy stuff which he wants to get rid of sooner rather then later. So I'm not sure if there is any point in looking this at VUB side?

Reply:
ok, well this is up to you.